src/common/com/intellij/plugins/haxe/ide/folding/HaxeFoldingBuilder.java

```diff
@@ -42,27 +47,29 @@
 
 public class HaxeFoldingBuilder implements FoldingBuilder {
 
+  private static final Key<Boolean> REGION_DEFINITION_KEY = new Key<>("RegionDefinition");
```

Comment:
Maybe we should use "HaxeRegionDefintion," just to keep out of the way of other plugins.  It just seems like something that will have a conflict sooner or later.